packages/react/src/errors.ts

```diff
@@ -31,3 +31,6 @@ export function isMagicLinkError(err: any): err is MagicLinkError {
 
 export const invalidStateError =
   'Invalid state. Feel free to submit a bug or reach out to support here: https://clerk.com/support';
+
+export const domainAsFunctionError =
+  'The usage of a function as domain is not supported in non-browser environments. Please contact support@clerk.com';
```

Comment:
:upside_down_face: Perhaps the "contact" part of the message is redundant?

Reply:
TBH this was requested, but I may have taken it a bit seriously 
cc @nikosdouvlis

Reply:
I suggested this error message as apart from the components, this is one of the few features we don't support on non-browser environment for now. This  error does not indicate missuse of our apis or an actual error - just a missing feature :)

Feel free to adjust accordingly or remove completely - no real issues here!

Reply:
I am thinking of changing this one to 
```typescript
export const unsupporterDomainFunctionInNonBrowser =
  'Unsupported usage of domain. The usage of domain as function is not supported in non-browser environments.';
```

Reply:
This is better, thanks Panteli